config/localRole.js

```diff
@@ -0,0 +1 @@
+module.exports = 'macuser';
```

Comment:
I saw this file in the gitignore, but it looks like it still got pushed up to github...

Reply:
Thanks, looking into it